### PR TITLE
[Bulk] validation-Odata-docs-invalidAuthor2

### PR DESCRIPTION
**Pull request by Roudabeh1348**
This PR fixes 16 issues across multiple categories in Odata-docs repo.

### CODE REVIEW

**Comment by Roudabeh1348** [comment]
@xuzhg @odero @Michael Pizzo The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. 
This will eliminate potential accessibility, security, and usability issues. 
This PR includes only build validation fixes and does not change other content. 
Please review within five business days and merge or comment in the PR with any changes you’d like to see. 
Thank you!

File: Odata-docs/odatalib/v7/type-annotation.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Override type annotation in serialization"
-description: ""
+description: "In ODataAnnotatable, there is a property called TypeAnnotation whose type is ODataTypeAnnotation. It is used to store the @odata.type annotation read from or written to the payload. "
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/type-annotation.md:3]:
```suggestion
description: "This section describes how to override type annotation during serialization"
```

File: Odata-docs/odatalib/v7/use-uri-parser.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Use ODataUriParser in odatalib v7"
-description: "Parse OData uri using OData Core APIs"
+description: "This post guides you through the URI parser for OData V4, which is released with ODataLib V6.0 and later."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/use-uri-parser.md:3]:
```suggestion
description: "This section describes how to parse OData uri using the ODataUriParser."
```

File: Odata-docs/overview.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title : "OData overview"
 description: "OData overview."
+author: madansr7
```
Comment (KenitoInc) [inline Odata-docs/overview.md:4]: Also include `ms.author`

File: Odata-docs/resources/roadmap.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ---
 title : "OData roadmap"
+description: "OData Version 4.01 adds various new features and removes a few restrictions. New OData 4.01 query features and simplified syntax can be supported as compatible extensions to OData 4.0 syntax."
```
Comment (KenitoInc) [inline Odata-docs/resources/roadmap.md:3]:
```suggestion
description: "This section describes the OData roadmap."
```

File: Odata-docs/spatial/create-instances.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 ---
 title: " Create spatial instances"
-description: "Create spatial instances"
+description: "Create spatial instances using Spatial APIs."
```
Comment (KenitoInc) [inline Odata-docs/spatial/create-instances.md:3]:
```suggestion
description: "This section describes how to create spatial instances using Spatial APIs."
```

File: Odata-docs/spatial/define-property.md [hunk 6]
```diff
@@ -1,7 +1,8 @@
 ---
 title: "Define spatial properties"
-description: "Define spatial properties in entity data models"
+description: "Define spatial properties in entity data models using EdmLib APIs."
```
Comment (KenitoInc) [inline Odata-docs/spatial/define-property.md:3]:
```suggestion
description: "This section describes how to define spatial properties in entity data models using EdmLib APIs."
```

File: Odata-docs/odatalib/v7/write-payload.md [hunk 7]
```diff
@@ -1,8 +1,8 @@
 ---
 title: " Write OData payload"
-description: "Write OData payload using OData Core APIs"
+description: "Write OData payloads including including service document, model metadata, entity set, entity, entity references, complex values, primitive values using OData Core APIs."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/write-payload.md:3]:
```suggestion
description: "This section describes how to write OData payloads including service document, model metadata, entity set, entity, entity references, complex values, primitive values using OData Core APIs."
```

**Review by KenitoInc — CHANGES_REQUESTED** [review]

File: Odata-docs/odatalib/v7/type-annotation.md (hunk 1 above)
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/type-annotation.md:3]: fixed.

File: Odata-docs/odatalib/v7/use-uri-parser.md (hunk 2 above)
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/use-uri-parser.md:3]: fixed.

File: Odata-docs/overview.md (hunk 3 above)
Comment (Roudabeh1348) [reply Odata-docs/overview.md:4]: fixed.

File: Odata-docs/resources/roadmap.md (hunk 4 above)
Comment (Roudabeh1348) [reply Odata-docs/resources/roadmap.md:3]: fixed.

File: Odata-docs/spatial/create-instances.md (hunk 5 above)
Comment (Roudabeh1348) [reply Odata-docs/spatial/create-instances.md:3]: fixed.

File: Odata-docs/spatial/define-property.md (hunk 6 above)
Comment (Roudabeh1348) [reply Odata-docs/spatial/define-property.md:3]: fixed.

**Comment by Roudabeh1348** [comment]
@habbes @KenitoInc the suggested revisions for descriptions were made to the mentioned articles in this PR.

**Review by KenitoInc — APPROVED** [review]
LGTM

File: Odata-docs/odatalib/v7/write-payload.md (hunk 7 above)
Comment (Roudabeh1348) [reply Odata-docs/odatalib/v7/write-payload.md]: fixed.

**Review by xuzhg — APPROVED** [review]

**Pull request merged**